packages/block-library/src/style.scss

```diff
@@ -14,6 +14,7 @@
 @import "./details/style.scss";
 @import "./embed/style.scss";
 @import "./file/style.scss";
+@import "./form-input/style.scss";
```

Comment:
Can we add this only when experiment is active?

Reply:
I don't think we can do that for CSS... 🤔

Reply:
Dumb question is dumb :)